radiation/radiation_cloud_generator.F90

```diff
@@ -539,8 +539,7 @@ subroutine sample_from_pdf_simd(this, fsd, cdf, x)
     use parkind1,              only : jprb
     use radiation_pdf_sampler, only : pdf_sampler_type
     implicit none
-#if defined(__GFORTRAN__) || defined(__PGI) || defined(__NEC__) || defined(__INTEL_LLVM_COMPILER)
-#else
+#if defined(__COMPILER_THAT_SUPPORTS_THIS_DIRECTIVE)
```

Comment:
Thanks for the suggestion, I agree that this would make more sense. I assume this directive has been added exclusively for ifort classic, so maybe simply guarding it like this would be the best approach:
```suggestion
#if defined(__INTEL_COMPILER) && !defined(__INTEL_LLVM_COMPILER)
```
For a single use of this directive, a build system-side handling seems a bit overkill to me.

Reply:
I trust you on the compiler that's supposed to support this. I pushed the suggestion.

And I also agree with you : it's best if we can avoid having to tweak the build system.